### DISCUSSION TITLE
customize the json data to badge

### DISCUSSION THREAD

**Discussion by nevespt** [upptime/upptime#532] [Q&A]
i wonder if i can change the colors output of
{"schemaVersion":1,"label":"uptime","message":"98.13%","color":"brightgreen"}
to an custom colors then is in a "green state"

**Comment by truxnell** [comment]
~~Also came here looking for this!  Ability to customize the badge fields would be nice, and i suspect not too bad a code change...~~

Apologies, if you are using shields.io like i was looking to do, you can do this with queries, no need for additions to this (awesome) project.
https://shields.io/endpoint

As an example with the docs url, you can do the following overrides like i did below by appending to the query string.
This changes
* Colour
* Label
* Adds a logo
* adds a badge style
![](https://img.shields.io/endpoint?color=blue&label=Google&logo=google&logoColor=red&style=for-the-badge&url=https%3A%2F%2Fraw.githubusercontent.com%2Fupptime%2Fupptime%2Fmaster%2Fapi%2Fgoogle%2Fuptime.json)
